src/settings-ui/Settings.UI/SettingsXAML/Views/VideoConference.xaml

```diff
@@ -43,90 +42,90 @@
                     <controls:SettingsExpander
                         x:Uid="VideoConference_SelectedCamera"
                         HeaderIcon="{ui:FontIcon Glyph=&#xE960;}"
                         IsExpanded="True">
                         <ComboBox
                             MinWidth="{StaticResource SettingActionControlMinWidth}"
-                            IsEnabled="{Binding Mode=TwoWay, Path=IsEnabled}"
-                            ItemsSource="{Binding CameraNames, Mode=OneTime}"
-                            SelectedIndex="{Binding Path=SelectedCameraIndex, Mode=TwoWay}" />
+                            IsEnabled="{x:Bind ViewModel.IsEnabled, Mode=TwoWay}"
+                            ItemsSource="{x:Bind ViewModel.CameraNames, Mode=OneTime}"
+                            SelectedIndex="{x:Bind ViewModel.SelectedCameraIndex, Mode=TwoWay}" />
                         <controls:SettingsExpander.Items>
                             <controls:SettingsCard x:Uid="VideoConference_CameraOverlayImagePathHeader">
                                 <StackPanel Orientation="Horizontal" Spacing="8">
-                                    <Button x:Uid="VideoConference_CameraOverlayImageBrowse" Command="{Binding Mode=OneWay, Path=SelectOverlayImage}" />
+                                    <Button x:Uid="VideoConference_CameraOverlayImageBrowse" Command="{x:Bind ViewModel.SelectOverlayImage, Mode=OneWay}" />
                                     <Button
                                         x:Uid="VideoConference_CameraOverlayImageClear"
-                                        Command="{Binding Mode=OneWay, Path=ClearOverlayImage}"
-                                        IsEnabled="{Binding Path=CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToBoolConverter}}" />
+                                        Command="{x:Bind ViewModel.ClearOverlayImage, Mode=OneWay}"
+                                        IsEnabled="{x:Bind ViewModel.CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToBoolConverter}}" />
                                 </StackPanel>
                             </controls:SettingsCard>
-                            <controls:SettingsCard Visibility="{Binding Path=CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToCollapsedConverter}}">
+                            <controls:SettingsCard Visibility="{x:Bind ViewModel.CameraImageOverlayPath, Mode=OneWay, Converter={StaticResource EmptyToCollapsedConverter}}">
                                 <Border
                                     MaxHeight="100"
                                     HorizontalAlignment="Right"
                                     CornerRadius="4">
                                     <Image
                                         x:Uid="VideoConference_CameraOverlayImageAlt"
-                                        Source="{Binding Mode=OneWay, Path=CameraImageOverlayPath}"
```

Comment:
This change crashes VCM page

Reply:
Thank, that's the one! How did you find out?

Reply:
good old "revert line by line" method :D